vcd/resource_vcd_vm_placement_policy.go

```diff
@@ -150,30 +157,45 @@ func sharedVcdVmPlacementPolicyRead(ctx context.Context, d *schema.ResourceData,
 		if policyName == "" {
 			return diag.Errorf("both Placement Policy name and ID are empty")
 		}
-		if pVdcId == "" {
-			return diag.Errorf("both Provider VDC ID and Placement Policy ID are empty")
-		}
-
 		method = "name"
 		queryParams := url.Values{}
-		queryParams.Add("filter", fmt.Sprintf("name==%s;pvdcId==%s", policyName, pVdcId))
-		filteredPoliciesByName, err := vcdClient.GetAllVdcComputePoliciesV2(queryParams)
+
+		var foundPolicies []*govcd.VdcComputePolicyV2
+		var err error
+		// Fetches the VM Placement Policy with Provider VDC information, intended for System admins
+		if vdcId == "" {
+			queryParams.Add("filter", fmt.Sprintf("%spolicyType==VdcVmPolicy;isSizingOnly==false;name==%s;pvdcId==%s", getVgpuFilterToPrepend(vcdClient, false), policyName, pVdcId))
+			foundPolicies, err = vcdClient.GetAllVdcComputePoliciesV2(queryParams)
+		} else {
+			var adminOrg *govcd.AdminOrg
+			// Fetches the VM Placement Policy with VDC information, intended for tenants
+			adminOrg, err = vcdClient.GetAdminOrg("") // Gets the Org from Provider config
```

Comment:
Thinking out loud

Should we have an optional `org` field at data source level as well so that people can specify a particular Org instead of the one coming from `provider` only? Can it break some user workflows if one cannot override a particular org here?

 If we had the field, we could use `GetAdminOrgFromResource`


cc @dataclouder

Reply:
Probably makes sense, if the Policy to fetch is in a VDC that belongs to another org.

Reply:
Changed!

Reply:
To leave history behind. We don't need to retrieve `org` or `vdc` structures at all here. We only need to put vdcId into request URL for placement policy lookup. The mandatory (for tenants) field `vdc_id` is sufficient to lookup placement policy